src/app/config/config.ts

```diff
@@ -235,6 +236,7 @@ const config: Config = {
   cookieSettings,
   isDev,
   useMockTwilio,
```

Comment:
nit: Could have just did `useMockPostmanSms: basicVars.core.useMockPostmanSms` instead of declaring a variable.